### DISCUSSION TITLE
Is the name a play on Parker Posey?

### DISCUSSION THREAD

**Discussion by vassudanagunta** [praetorian-inc/noseyparker#36] [General]
Which one of you has a crush on her?

**Comment by bradlarsen** [comment]
😆 

No, the name is from [British English slang](https://www.merriam-webster.com/dictionary/nosy%20parker):

> Nosy parker, a person who is too interested in what other people are doing : a nosy person

See <https://www.phrases.org.uk/meanings/nosy-parker.html> for more.
